tests/Endpoints/ClientTest.php

```diff
@@ -10,12 +12,6 @@
 
 class ClientTest extends TestCase
 {
-    protected function setUp(): void
-    {
-        parent::setUp();
-        $this->client->deleteAllIndexes();
-    }
-
     public function testGetAllIndexesWhenEmpty()
```

Comment:
Maybe we can add return typehint for each test method even if it's `: void`?

Reply:
This seems useless, I've added the `:void` to setUp because that's the method signature.